internal/util/reports.go

```diff
@@ -13,7 +13,7 @@ var (
 	reportsHost  = "kong-hf.konghq.com"
 	reportsPort  = 61833
 	pingInterval = 3600
-	tlsConf      = tls.Config{MinVersion: tls.VersionTLS13}
+	tlsConf      = tls.Config{MinVersion: tls.VersionTLS12, MaxVersion: tls.VersionTLS12} // nolint:gosec
```

Comment:
Can you please help me to better understand this change?

Reply:
It's what Google had certified, and the library is limited to it: https://github.com/golang/go/issues/46898

As TLS 1.2 is still secure and we shouldn't need its performance optimizations (we don't make that many connections they're not particularly latency-sensitive) so I thought it made more sense to drop this across the board rather than try to handle it separately for the two builds.